src/hooks/useTrailPaths.ts

```
@@ -0,0 +1,106 @@
+import { useState, useEffect, useCallback } from 'react';
+import type { GeoJSONFeatureCollection, TrailPathData } from '../types/trail';
+import { getTrailPaths } from '../utils/api';
+import { convertFeatureCollectionToTrailPathsSafe } from '../utils/trailConverter';
+
+interface UseTrailPathsReturn {
```

ESLint

[error] 'UseTrailPathsReturn' is defined but never used.

(@typescript-eslint/no-unused-vars)